includes/class-sensei-modules.php

```diff
@@ -2492,8 +2492,7 @@ public function filter_terms_by_owner( $terms, $user_id ) {
 			}
 		}
 
-		return $users_terms;
-
+		return apply_filters( 'sensei_filter_module_terms_by_owner', $users_terms, $terms, $user_id );
```

Comment:
Could we add a doc comment for this new filter?

Reply:
Done [34b8f15](https://github.com/Automattic/sensei/pull/5345/commits/34b8f15d5e8ce31d860b1e3ddd1bd9849591909b)